refaster-support/src/test/java/tech/picnic/errorprone/refaster/matchers/IsEmptyTest.java

```diff
@@ -100,36 +115,195 @@ void matches() {
             "    return new Vector<>(ImmutableList.of(1));",
             "  }",
             "",
-            "  ImmutableList<Integer> negative16() {",
-            "    return ImmutableList.of(1);",
+            "  Stream<Integer> negative16() {",
+            "    return Stream.of(1);",
             "  }",
```

Comment:
Here and below: some tests where moved around to match the new structure of the main code. This made it easier for me to validate that I covered all test cases.